src/app/content/studyGuides/hooks/loadMore.ts

```diff
@@ -40,11 +41,14 @@ export const loadMore = async(services: MiddlewareAPI & AppServices, pageSize?:
 export type LoadMoreResponse = ReturnType<typeof loadMore>;
 
 export const hookBody: ActionHookBody<
-  typeof actions.setDefaultSummaryFilters |
+  typeof locationChange |
   typeof actions.setSummaryFilters |
-  typeof actions.updateSummaryFilters |
   typeof actions.loadMoreStudyGuides
 > = (services) => async() => {
+  const {book, page} = bookAndPage(services.getState());
+
+  if (!book || !page) { return; }
+
```

Comment:
I think so, because it is now attached to the `locationChange` which happens before `book` or `page` is loaded
This probably is happening only on the testing env